modules/stanford_paragraph_video/config/install/core.entity_view_display.paragraph.stanford_video.default.yml

```diff
@@ -0,0 +1,55 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.stanford_video.field_p_video_url
+    - paragraphs.paragraphs_type.stanford_video
+  module:
+    - ds
+    - field_formatter_class
+    - video_embed_field
+third_party_settings:
+  ds:
+    layout:
+      id: pattern_section_header
```

Comment:
This seems like cheating to me. We should create a generic centered option for items like this.